frontend/public/components/graphs/helpers.ts

```diff
@@ -8,10 +8,12 @@ export enum PrometheusEndpoint {
   QUERY_RANGE = 'api/v1/query_range',
 }
 
+export const getPrometheusQeuryEndTimestamp = () => Date.now();
```

Comment:
If you think that this is necessary and using just `Date.now()` in `adjustDurationForVMI` is not enough then I would be more inclined to specify the `endTime: Date.now()` in dashboard actions https://github.com/openshift/console/blob/master/frontend/public/actions/dashboards.ts#L104 and reference the function for there and keep this file as was

Reply:
The point is to reuse this "logic" of setting `Date.now()` as the end of the period even within the `adjustDurationForStart()`. If we decide to change that, it will be clear what other logic depends on that.

Reply:
But its just a default value and actions/dashboards can change it at some point which would break the `adjustDurationForVMI`. So its better for `adjustDurationForVMI` to depend on the value passed from `actions/dashboards` rather then the default value of in `graph/helpers`